src/modules/httptrap.c

```
@@ -457,6 +458,12 @@ rest_get_json_upload(mtev_http_rest_closure_t *restc,
 
   content_length = mtev_http_request_content_length(req);
   rxc = restc->call_closure;
+  rxc->check = noit_poller_lookup(rxc->check_id);
+  if (!rxc->check) {
+    *complete = 1;
```

should we report an error, here?